PFR/endgame.lean

```diff
@@ -41,8 +41,9 @@ variable (p : refPackage Ω₀₁ Ω₀₂ G)
 variable {Ω : Type*} [mΩ : MeasureSpace Ω] [IsProbabilityMeasure (ℙ : Measure Ω)]
 
 variable (X₁ X₂ X₁' X₂' : Ω → G)
+  (hX₁ : Measurable X₁) (hX₂ : Measurable X₂) (hX₁' : Measurable X₁') (hX₂' : Measurable X₂')
```

Comment:
I added these measurability conditions and the `MeasurableAdd₂ G` class above since they seemed to be necessary to apply `condMutualInformation_comm`. Does that seem right?

Reply:
Yes, I think we will need measurability hypotheses like this throughout the paper (of course in the end they will all be automatic because all our spaces are finite with the discrete sigma alegbra).  In retrospect we probably should have worked with a "bundled" random variable class in which the measurability hypotheses were built into the class, but we're so deep into the project right now that it may be too problematic to change it for now.